o-fish-ios/Views/ReportFlow/ChooseScreens/ChooseViolationsView.swift

```diff
@@ -13,7 +13,7 @@ struct ChooseViolationsView: View {
         if Settings.shared.menuData.violationCodes.count == Settings.shared.menuData.violationDescriptions.count {
             for index in Settings.shared.menuData.violationCodes.indices {
                 let violation = ViolationPickerData(
-                    caption: "California",  // TODO: Remove once agreed with design
+                    caption: "",
```

Comment:
Are we need caption at all? Investigate oportunity to remove it.

Reply:
Not for violations